web-ui/src/main/resources/catalog/templates/editor/directory.html

```diff
@@ -184,39 +184,37 @@ <h5 class="btn-group">
                           {{'directoryEntryValidate' | translate}}</a>
                         </li>
                         <li><a href ng-click="rejectEntry(e)">
-                          <i class="fa fa-ban fa-fw" />&nbsp;
+                          <i class="fa fa-ban fa-fw text-danger" />&nbsp;
                           {{'directoryEntryReject' | translate}}</a>
                         </li>
                         <li><a href ng-click="delEntry(e)">
-                          <i class="fa fa-times fa-fw text-danger" />&nbsp;
+                          <i class="fa fa-trash fa-fw" />&nbsp;
```

Comment:
I think I remember past discussionsss where we decided to only use fa-times + text-danger in all the app for removing something. So we should avoid fa-trash here I think.